extensions/amp-access/0.1/test/test-amp-access-server-jwt.js

```diff
@@ -581,7 +581,7 @@ describes.realWin('AccessServerJwtAdapter', {amp: true}, env => {
       beforeEach(() => {
         jwt = {
           'exp': Math.floor((Date.now() + 10000) / 1000),
-          'aud': 'ampproject.org',
+          'aud': 'amp.dev',
```

Comment:
this needs to be ampproject.org (for the whole file)

Reply:
fixed